src/src/hilell/hw/FileManager.java

```diff
@@ -0,0 +1,111 @@
+package hilell.hw;
+
+import java.io.IOException;
+import java.nio.file.*;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class FileManager {
+    private static final Logger logger = Logger.getLogger(FileManager.class.getName());
+    private static Path currentDirectory;
+
+    public static void main(String[] args) {
+        currentDirectory = Paths.get("user.home");
```

Comment:
Are you sure this will return the root path to your C drive?
I suppose it won't. I meant this _String rootPath = System.getProperty("user.home");
currentDirectory = Paths.get(rootPath);_
Try it first, in case you have questions, do not hesitate and ask in LMS